Automattic/woocommerce-payments
# Checking if wcpayPaymentRequestPayForOrderParams before using it

Fixes #7996

#### Changes proposed in this Pull Request

This PR adds a check for wcpayPaymentRequestPayForOrderParams before using it so no error appears in the console. The checkout works fine already, we're just not triggering an error anymore.

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/9b83cade-4de0-4a8c-bb40-47a4183b074e)


#### Testing instructions

- Create a pending payment order for an existing user.
- Copy the link for the Customer payment page into a new incognito/private browser window in Chrome or Safari (or a browser when not logged in to the site).
- The wcpayPaymentRequestPayForOrderParams localization wcpayPaymentRequestPayForOrderParams param will not be defined, but there should be no error in the console

-------------------


**Post merge**
